sdks/python/featurehub-python-sdk/featurehub_client_test.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+import featurehub_client
+from unittest.mock import MagicMock
+
+
+class FeatureHubClientTest(unittest.TestCase):
+
+    def test_make_fh_client_object(self):
+        repo = featurehub_client.FeatureHubRepository()
+        fh_client = featurehub_client.FeatureHubClient("http://localhost", ["abc", "123"], repo)
+        # testing
+        # fh_client = featurehub_client.FeatureHubClient("https://kd0gwa.demo.featurehub.io", ["default/190017b3-3e4d-4c68-9805-d52c2b597fe0/hrn6TUohWtUFIvlPCar6RVYkZ5fZH9*3VnmTK5Wp47EQiG6x38n"
+        # ,'default/29517115-da60-4b64-99db-9da017561edd/7NZq23UuFEdmpcwxx7lkKSH0QH1EIS*dmGCHaOErbXyMLfzHCe0'], repo)
+
+        self.assertIsInstance(fh_client, featurehub_client.FeatureHubClient)
+
+    def test_is_ready_func(self):
+        repo = featurehub_client.FeatureHubRepository()
+        self.assertFalse(repo.is_ready())
+        repo.ready = True
```

Comment:
ready should be read only, if you send it a notify with some data, it should go into ready and have updated the features.